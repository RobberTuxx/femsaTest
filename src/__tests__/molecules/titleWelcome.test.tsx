import React from 'react';
import renderer from 'react-test-renderer';
import TitleWelcome from "../../components/molecules/titleWelcome";
it('render score', ()=> {
    const component = renderer.create(<TitleWelcome/>).toJSON()
    expect(component).toMatchSnapshot()
})
it('has 0 child', ()=> {
    const component = renderer.create(<TitleWelcome/>).toJSON()
    expect(component.children.length).toBe(2)
})
