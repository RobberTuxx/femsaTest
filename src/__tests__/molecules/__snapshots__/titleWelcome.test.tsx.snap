// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render score 1`] = `
<View>
  <Text
    style={
      {
        "color": "#020202",
        "fontFamily": "Avenir",
        "fontSize": 31.5,
        "fontWeight": "800",
      }
    }
  >
    hello
  </Text>
  <Text
    style={
      {
        "color": "#020202",
        "fontFamily": "Avenir",
        "fontSize": 25,
        "fontWeight": "400",
      }
    }
  >
    hello
  </Text>
</View>
`;
